buidl/bcur.py

```diff
@@ -0,0 +1,253 @@
+from buidl.bech32 import (
+    cbor_encode,
+    cbor_decode,
+    bc32encode,
+    bc32decode,
+    uses_only_bech32_chars,
+)
+from buidl.helper import is_intable
+
+from binascii import a2b_base64, b2a_base64
+from math import ceil
+
+import hashlib
+
+
+# BCUR Exceptions
+
+
+class INCONSISTENT_BCUR_STRING(RuntimeError):
```

Comment:
Not a huge fan of all-caps exceptions. Generally, I've seen camel-cased exceptions ending with "Error" or "Exception" so you would have InconsistentStringError (or ValueError?), InvalidChecksumError, InvalidEncodingError (SyntaxError?) and InvalidStringError (or SyntaxError?). You can use some built in ones if they make sense, but I would like to see the classes follow naming conventions of the rest of Python.

Reply:
Oof! clearly I haven't been writing a lot of python lately. Good catch.